### PR TITLE
Java Examples for onnx and MNIST for TF 

### PR DESCRIPTION
**Pull request by venkat-nidrive**
Java Example docs created for illustrate the konduit serving framework for model step and python step in onnx and mnist.

document for bert is moved to subfolder to make same level as mnist.

### CODE REVIEW

**Review by ShamsUlAzeem — CHANGES_REQUESTED** [review]
Make sure you update the summary.md file like I've done here: https://github.com/KonduitAI/konduit-serving-docs/pull/6/files so the newly added files are picked up by gitbook

**Comment by ShamsUlAzeem** [comment]
@venkat-nidrive a few changes to do here. I've left comments

**Comment by venkat-nidrive** [comment]
> Make sure you update the summary.md file like I've done here: https://github.com/KonduitAI/konduit-serving-docs/pull/6/files so the newly added files are picked up by gitbook

Updated summary.md file with latest java examples .

File: examples/java/dl4j_java.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 description: >-
-  This page documents two ways to create Konduit Serving configurations with the Java SDK:  
+  This documents illustrates how to create Konduit Serving configurations with the Java SDK:
```
Comment (ShamsUlAzeem) [inline examples/java/dl4j_java.md]: typo: should be `This document`

**Review by ShamsUlAzeem — CHANGES_REQUESTED** [review]
@venkat-nidrive minor nits

File: examples/java/dl4j_java.md (hunk 1 above)
Comment (venkat-nidrive) [reply examples/java/dl4j_java.md]: Typo issue fixed.

**Pull request merged**